.circleci/config.yml

```diff
@@ -215,21 +245,42 @@ jobs:
           paths:
             - skupper-manifest
 
+  generate-operator-bundle:
+    executor:
+      name: go_cimg
+    steps:
+      - checkout
+      - go/mod-download-cached
+      - kubectl-install
+      - operator-sdk-install
+      - run: make generate-bundle
+      - persist_to_workspace:
+          root: .
+          paths:
+            - operator-bundle
+
   generate-skupper-setup:
     executor:
       name: go_cimg
     steps:
       - checkout
       - go/mod-download-cached
-      - run:
-          name: Install Helm CLI
-          command: curl https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3 | bash
+      - kubectl-install
+      - helm-install
+      - oras-install
+      - run: make generate-skupper-helm-chart
       - run: make generate-skupper-deployment-cluster-scoped
       - run: make generate-skupper-deployment-namespace-scoped
       - run: make generate-network-observer-generic
       - run: make generate-network-observer-openshift
       - run: mkdir skupper-setup
       - run: cp ./*.yaml skupper-setup
+      - run: make pack-skupper-helm-chart
+      - run: cp ./skupper-*.tgz skupper-setup
+      - run: helm registry login -u ${QUAY_LOGIN} -p ${QUAY_PASSWORD} quay.io
+      - run: helm push skupper-*.tgz oci://quay.io/skupper/helm
```

Comment:
@c-kruse do you want me to include the commands that push the network observer chart in this pr?